src/components/history/HistoryPagination.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import {injectIntl} from "react-intl";
+import withI18n from "../../i18n/withI18n";
+import {ACTIONS_PER_PAGE, PAGINATION_DIRECTION } from "../../constants/DefaultConstants";
+import PropTypes from "prop-types";
+
+let HistoryPagination = (props) => (
+    <nav className="d-flex justify-content-center">
```

Comment:
I think flexbox is much better than bootstrap but for consistency with the rest of the app I assume bootstrap should be used